src/ObjectUtils/CoffFile.cpp

```diff
@@ -65,6 +68,22 @@ CoffFileImpl::CoffFileImpl(std::filesystem::path file_path,
   }
 }
 
+ErrorMessageOr<void> CoffFileImpl::Initialize() {
+  for (const llvm::object::SectionRef& section_ref : object_file_->sections()) {
```

Comment:
I think this deserves a comment, that there is only one text section expected in a Coff file.

Reply:
Added.